android-interop-testing/build.gradle

```diff
@@ -47,10 +47,10 @@ android {
 }
 
 dependencies {
-    implementation 'com.android.support:appcompat-v7:26.1.0'
-    implementation 'com.android.support:multidex:1.0.3'
-    implementation 'com.android.support:support-annotations:27.1.1'
-    implementation 'com.google.android.gms:play-services-base:15.0.1'
+    implementation 'androidx.appcompat:appcompat:1.3.0-alpha02'
```

Comment:
Why use alpha versions?

Reply:
Shall we avoid using alpha versions? I was just upgrade everything to versions as latest as possible.

Reply:
Changed to use stable versions.

Reply:
Yeah, we prefer stable versions.